pygenesys/tests/test_eia_data.py

```diff
@@ -0,0 +1,204 @@
+from pygenesys.data.eia_data import *
+import pytest
+from datetime import date
+import pandas as pd
+
+
+def test_get_date():
+    """
+    Tests the get_date function.
+    """
+
+    today = date.today().strftime("%B %d, %Y")
+
+    today = today.split(' ')
+
+    m, d, y = get_date()
+
+    assert m == today[0]
+    assert d == today[1]
+    assert y == int(today[2])
+
+    return
```

Comment:
Are these return values necessary? If not, consider removing them.

Reply:
Could you elaborate on why removing them is recommended? Does it just reduce the lines of code?

Reply:
That's one reason. Additionally, we are checking these functions with `pytest` via the `assert` statements. The return statements don't really serve a purpose in the test suite. Did you have a specific goal in mind when including them?

Reply:
I find it easier to read code when there's a definite return statement. Otherwise, there is a tendency for everything to bleed together. For example, class methods often don't have return statements. My MO is that every function/method should have a return statement, even if it's empty.